reveal/run.js

```diff
@@ -107,7 +107,16 @@ eslint no-param-reassign: [
         { src: 'reveal/plugins/zenika-footer/zenika-footer.js', condition() { return !!document.body.classList; } },
         { src: `${config.revealModule}plugin/math/math.js`, async: true },
       ],
-    });
+    };
+
+
+    fileExists('/framework/index.js')
+        .then((script) => {
+          /* eslint no-eval: "off", no-undef: "off" */
```

Comment:
I suggest to only disable the next line
```suggestion
          // eslint-disable-next-line no-eval no-undef
```